### ISSUE TITLE
changelog for a release-pipeline

### ISSUE THREAD

**Issue by robstoll** [loewenfels/dep-graph-releaser#72]
Depending on how many projects a pipeline consists of, one loses the overview effect and thus would like to have kind of a changelog for a given pipeline in text format/CSV. I would imagine something as follows, taking https://loewenfels.github.io/dep-graph-releaser/#./release.json as an example:
``` 
Project; Release Version; Next Dev Version; Other Commands
dgr-4;   1.3.0;            1.4.0-SNAPSHOT;
dgr-3;   0.13.0;          0.14.0-SNAPSHOT; update dgr-4
dgr-3-b: 0.13.0;          0.14.0-SNAPSHOT; update dgr-4
dgr-3-c: 0.13.0;          0.14.0-SNAPSHOT; update dgr-4
dgr-5;   1.2.0rc4;      1.2.0rc5-SNAPSHOT; update dgr-4
dgr-2;   0.1.55;          0.1.56-SNAPSHOT; update dgr-5; update dgr-3
dgr-1;   0.1.87;          0.1.87-SNAPSHOT; update dgr-2; update dgr-3b; update dgr-3a; update dgr-3-c
```

I would place a new button under Menu Tools. In case we have more of such reports we could introduce an own menu entry.

**Comment by robstoll** [comment]
Excel and CSV 😭 provide two buttons one which provides a real csv for text editors and another one for excel where versions are wrapped into ="..." otherwise it is likely that they get treated as dates in excel. For instance, 1.5.0 is 1st of May 2000

**Comment by robstoll** [comment]
resolved with c4551c5
